packages/esm-patient-forms-app/src/clinical-form-action-button.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ClinicalFormActionButton from './clinical-form-action-button.component';
+import { render, screen } from '@testing-library/react';
+import * as mockEsmFramework from '@openmrs/esm-framework';
+
+describe('<ClinicalFormActionButton/>', () => {
+  test('should display clinical form action button on tablet view', () => {
+    spyOn(mockEsmFramework, 'useLayoutType').and.returnValue('tablet');
+    render(<ClinicalFormActionButton />);
+    const clinicalActionButton = screen.getByRole('button', { name: /Clinical form/ });
+
+    expect(clinicalActionButton).toHaveTextContent('Clinical form');
```

Comment:
This is redundant with the `getByRole` call. I think the `expect...toBeInTheDocument` is enough.